eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -495,6 +495,42 @@
           "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
           "old": ".*? com\\.azure\\.resourcemanager\\.network\\.models.*",
           "justification": "Jackson annotation changed."
+        },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
+          "old": ".*? com\\.azure\\.resourcemanager\\.cosmos\\.models.*",
+          "justification": "Jackson annotation changed."
+        },
+        {
+          "code": "java.method.removed",
+          "old": "method java.lang.Integer com.azure.resourcemanager.cosmos.models.ServiceResourceCreateUpdateParameters::instanceCount()",
+          "justification": "The property flatten disabled due to model changed to polymorphic."
```

Comment:
We should mention these breaking changes in changelog, e.g.`ServiceResourceCreateUpdateProperties` is no longer flattened in `ServiceResourceCreateUpdateParameters`, as it's now polymorphic.

Reply:
Added breaking changes to `CHANGELOG.md` in the new version.